src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -21,7 +21,9 @@ namespace System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests
 {
     public static partial class DecryptTests
     {
-        [Fact]
+        public static bool IsNet462OrNewer() => (!PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx462OrNewer());
```

Comment:
That just happens to be the in-test name for the variables.  The difference between them and the RSATransferCapi certs is that the RSATransfer ones use CNG for key storage and the Capi ones use CAPI.  So "Cng" is really the behavioral difference we're probing on.